### PR TITLE
Make docker toolchain and tighten tests

### PR DESCRIPTION
**Pull request by jbbjarnason**

### CODE REVIEW

File: containers/start-dbus.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+dbus-daemon --system
+
+sed -i 's|deny own=|allow own=|g' /usr/share/dbus-1/system.conf
+sed -i 's|deny send_type="method_call"|allow send_type="method_call"|g' /usr/share/dbus-1/system.conf
+dbus-send --system --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig
```
Comment (omarhogni) [inline containers/start-dbus.sh:6]: Check this out, dont need to reload if we set the settings first.

Comment (omarhogni) [reply containers/start-dbus.sh:6]: sed can be done in container file

**Review by omarhogni — CHANGES_REQUESTED** [review]
Very good change

File: containers/start-dbus.sh (hunk 1 above)
Comment (jbbjarnason) [reply containers/start-dbus.sh]: changed

**Pull request merged**